pkg/addon/configpolicy/manifests/managedclusterchart/values.yaml

```diff
@@ -14,6 +14,9 @@ args:
   clientBurst: 45
 hubKubeConfigSecret: config-policy-controller-hub-kubeconfig
 
+operatorPolicy:
+  disabled: true
```

Comment:
Disabled by default? Is that correct?

Reply:
Also, this might need to be a string, looking at some other parameters?

Reply:
Oops, I had changed it from an "enabled" thing to a "disabled" thing and missed this. Funny enough, while I was doing that I thought "this might be confusing..." 😆

Reply:
I was thinking exactly the same thing! 😂  But it's really only confusing if you dig beyond the annotation, so for the end user hopefully it'll be okay...

Reply:
I think the string thing was wrong, but actually this also isn't being exposed as a custom annotation, is it?

Reply:
Missed this comment. I'm not sure what you mean exactly... there is a custom annotation to set this... it's called "disable", but if you set it to `false`, it will override the other logic, and enable it. If it's unset, the usual logic will be preserved.

Reply:
Right, I was off-base. I was a little confused because it wasn't in the `userArgs` struct, but I think that's fine since it's wiping out the entire flag/argument rather than just modifying it. And it's technically disabling an entire controller, so it's probably clearer in its own "operatorPolicy" object.